config-linux.md

```diff
@@ -171,12 +176,15 @@ The cgroups will be created if they don't exist.
    "cgroupsPath": "/myRuntime/myContainer"
 ```
 
-`cgroupsPath` can be used to either control the cgroups hierarchy for containers or to run a new process in an existing container.
+`cgroupsPath` can be used to either control the cgroup hierarchy for containers or to run a new process in an existing container.
 
 You can configure a container's cgroups via the `resources` field of the Linux configuration.
 Do not specify `resources` unless limits have to be updated.
 For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
 
+A runtime MUST at least use the minimum set of cgroup controllers required to fulfil the `resources` settings.
+However, a runtime CAN decide to not attach to any additional cgroup controllers supported by the system.
```

Comment:
On Fri, Jun 10, 2016 at 09:07:45PM -0700, Aleksa Sarai wrote:
> The reason I moved it here is because the `resources` section isn't
> mentioned before the examples section. I can move both parts above
> the examples though.

Yeah, I think you should move all this non-example stuff up out of the
example subsection.